reaktive/src/wasmJsMain/kotlin/com/badoo/reaktive/scheduler/JsFunctions.kt

```diff
@@ -0,0 +1,15 @@
+package com.badoo.reaktive.scheduler
+
+internal actual fun jsSetTimeout(task: () -> Unit, delayMillis: Int): Int =
```

Comment:
I afraid we don't have tests for JS MainScheduler currently. Can we test at least manually (in the sample app) that both cases work fine in Wasm? And I will think later if we can add tests for this case (it's a bit tricky because of the single-threaded environment).

```kotlin
fun test1() {
    val executor = mainScheduler.newExecutor()
    executor.submit(delay = 1.seconds) {
        println("Hit!")
    }
}

fun test2() {
    val executor = mainScheduler.newExecutor()
    executor.submit(delay = 1.seconds, period = 1.seconds) {
        println("Hit!")
    }
}
```

Reply:
I've added two new buttons to browser sample project which will allow to test this behavior.
Also' please take a look on the test I've made. I think it's covering this usecases. https://github.com/badoo/Reaktive/pull/771/commits/0ac05f97254cace523ac4f5d3b1606e2917b9630